schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-drawer-container>
+  <mat-drawer mode="side" #drawer [opened]="!isHandset">
+    <mat-toolbar color="primary">Menu</mat-toolbar>
+    <mat-nav-list>
+      <a mat-list-item href="#">Link 1</a>
+      <a mat-list-item href="#">Link 2</a>
+      <a mat-list-item href="#">Link 3</a>
+    </mat-nav-list>
+  </mat-drawer>
+  <mat-drawer-content>
+    <mat-toolbar color="primary">
+      <button type="button" mat-icon-button (click)="drawer.toggle()" *ngIf="isHandset">
+        <mat-icon aria-label="Side nav toggle icon">menu</mat-icon>
```

Comment:
The `aria-label` should go on the button instead of the icon and should describe the button's action rather than describe the icon.